provider/builtin/provider.go

```diff
@@ -181,9 +181,24 @@ func (p *builtinProvider) Evaluate(cap string, conditionInfo []byte) (lib.Provid
 			xmlFiles = append(xmlFiles, xhtmlFiles...)
 		} else if len(cond.XML.Filepaths) == 1 {
 			// Currently, rendering will render a list as a space seperated paths as a single string.
-			xmlFiles = strings.Split(cond.XML.Filepaths[0], " ")
+			patterns := strings.Split(cond.XML.Filepaths[0], " ")
+			for _, pattern := range patterns {
+				files, err := findFilesMatchingPattern(p.config.Location, pattern)
+				if err != nil {
```

Comment:
This bit seems odd, if there is an error then we add the pattern as a file? Can we add a comment on why?

Reply:
my thinking was if something broke unintentionally it just goes back to the default behavior of trusting user input and running the rule

Reply:
Yeah probably is the safest thing to do, I wonder can we add a debug log here I assume that we are going to be tracking down an obscure bug in the future :)

Reply:
done